manticore/ethereum/manticore.py

```diff
@@ -572,6 +579,78 @@ def make_symbolic_arguments(self, types):
         # FIXME this is more naive than reasonable.
         return ABI.deserialize(types, self.make_symbolic_buffer(32, name='INITARGS', avoid_collisions=True))
 
+    def json_create_contract(self, jfile, owner=None, name=None, contract_name=None, balance=0, gas=None, network_id=None):
+        """ Creates a solidity contract based on a truffle json artifact.
+
+            :param jfile: truffle json artifact
```

Comment:
Is there any chance we link or point to some source/docs regarding on how a truffle json artifact looks like?

Or to our own tests/examples.

Reply:
https://github.com/trailofbits/manticore/pull/1376#pullrequestreview-215730566